asset/style.scss

```diff
@@ -556,8 +555,15 @@ footer {
   // Built-in name ("this", etc.).
   .nb                     { color: hsl(200, 100%, 45%); }
 
+  .code-wrapper {
+    padding: 14px;
+    display: inline-block;
+    min-width: 100%;
```

Comment:
Of course, if my description made complete sense, we wouldn't need this `min-width` except that we do, for reasons I can't explain.